client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -135,13 +134,13 @@ class SiteTools extends Component {
 						isWarning
 					/>
 				}
-				{ showManageConnection &&
+				{ showDisconnectSite &&
```

Comment:
I doubt this is where you want to start your flow from.

Let's leave the **Manage your connection** link where it is, and rather start the disconnection flow from the `DisconnectSiteLink` in **Manage Connection**.